doc/en/components/grids/_shared/paging.md

```diff
@@ -57,7 +57,7 @@ The following example represents `{ComponentName}` pagination and exposes the op
 <!-- React -->
 ```tsx
 <{ComponentSelector} className="gridSize" height="500px" width="100%">
-    <IgrPaginator key="paginator" perPage="10">
+    <IgrPaginator key="paginator" perPage={10}>
```

Comment:
`key` is no longer needed. Please remove it from every react snippet

Reply:
removed